.github/workflows/CI.yml

```diff
@@ -37,7 +37,36 @@ jobs:
       - uses: actions/checkout@v3
       - name: Update Rust
         run: rustup update stable && rustup default stable
-      - run: cargo test --all-features
+      - name: Setup Docker (Mac)
+        if: ${{'macos-latest' == matrix.os}}
+        run: |
+          brew update
+          brew install docker
+          brew install qemu
+          brew install colima
+          brew install lima-additional-guestagents
+          # M1 and M2 Macs that the Github Actions supplies don't support nested virtualization. This forces x86 emulation to get around this
+          #--mount /private is necessary here in order to give docker access to mac's temp directory for tests
+          colima start --mount /private --memory 4 --network-address --arch x86_64
+          echo "DOCKER_HOST=unix:///Users/runner/.colima/docker.sock" >> $GITHUB_ENV
```

Comment:
I'm definitely going to copy the changes here to the `wdl` repository for `wdl-engine` tests.

Reply:
Awesome! :) Hopefully it runs reasonably given that it is fully emulated